added name in contriutors.md

SayamJain1 opened the pull request:
Hi I added my name in the contributors.md

SayamJain1 committed:
added name in contriutors.md

Umesh-01 reviewed (commented):
@SayamJain1 add your name and github profile link only, don't do the changes to other profiles..

SayamJain1 committed:
added name in contributor.md

SayamJain1 commented:
hey please preview my pull request is it right?

SayamJain1 commented:
umesh i just added the file again review it if i did something wrong please
guide me its my first contribution.

On Sat, Nov 13, 2021 at 11:25 PM Umesh Singh ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
>
> @SayamJain1 <https://github.com/SayamJain1> add your name and github
> profile link only, don't do the changes to other profiles..
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/firstcontributions/first-contributions/pull/45097#pullrequestreview-805389998>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AV7DKYP6C322FEMMDKITWLLUL2RCLANCNFSM5H6EO3FA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

SayamJain1 commented:
ohhh i got this i think i changed other contributors contribution because i
have this extension in my vs code that align code, So how can i wipe all
mess that i made?

On Sun, Nov 14, 2021 at 8:01 AM Sanyam Jain ***@***.***>
wrote:

> umesh i just added the file again review it if i did something wrong
> please guide me its my first contribution.
>
> On Sat, Nov 13, 2021 at 11:25 PM Umesh Singh ***@***.***>
> wrote:
>
>> ***@***.**** commented on this pull request.
>>
>> @SayamJain1 <https://github.com/SayamJain1> add your name and github
>> profile link only, don't do the changes to other profiles..
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/firstcontributions/first-contributions/pull/45097#pullrequestreview-805389998>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AV7DKYP6C322FEMMDKITWLLUL2RCLANCNFSM5H6EO3FA>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>>
>
